lib/iris/tests/runner/_runner.py

```diff
@@ -143,12 +143,6 @@ def run(self):
             regexp_pat,
             "--process-timeout=180",
         ]
-        try:
-            import gribapi  # noqa
-        except ImportError:
-            args.append("--exclude=^grib$")
-        if self.stop:
```

Comment:
You need to bring back the final 2 lines of the 6 you removed!

Reply:
Ooops
Good spot!